src/psyclone/domain/lfric/lfric_extract_driver_creator.py

```diff
@@ -971,7 +966,7 @@ def write_driver(self, nodes, input_list, output_list,
         '''This function uses the ``get_driver_as_string()`` function to get a
         a stand-alone driver, and then writes this source code to a file. The
         file name is derived from the region name:
-        "driver-"+module_name+"_"+region_name+".f90"
+        "driver-"+module_name+"_"+region_name+".F90"
```

Comment:
Have you switched to .F90 because that's what one or more of the modules use?

Reply:
No, because we are now inlining files with preprocessor directives (constants_mod.F90), and they don't compiler properly as .f90 (without additional options that is).

Reply:
As discussed in the telco: I've added a makefile target to preprocess all .F90 files in the LFRic infrastructure directory, and also added the preprocessed files. This means we can now parse all files.
Note thought that I was WRONG stating the the LFRic build system pre-processes first - it pre-processes implicitly when computing the dependencies, but these files are neither stored not used. So PSyclone is running on .F90 files (just typically the infrastructre files are not handled by PSyclone, so no problem - but with driver creation this will become a problem)

Reply:
That's fine for now. I've separately emailed Matthew about this as PSyclone should always operate on pre-processed files.